### PR TITLE
add rank_fields to rerank sdk

### PR DESCRIPTION
**Pull request by lfayoux**

### CODE REVIEW

File: cohere_aws/client.py [hunk 1]
```diff
@@ -387,13 +387,15 @@ def rerank(self,
...
-               max_chunks_per_doc: Optional[int] = None) -> Reranking:
+               max_chunks_per_doc: Optional[int] = None,
+               rank_fields: Optional[List[str]] = None) -> Reranking:
         """Returns an ordered list of documents oridered by their relevance to the provided query
         Args:
             query (str): The search query
             documents (list[str], list[dict]): The documents to rerank
             top_n (int): (optional) The number of results to return, defaults to return all results
             max_chunks_per_doc (int): (optional) The maximum number of chunks derived from a document
+            rank_fields (list[str]): (optional) The fields used for reranking
```
Comment (leilacc) [inline cohere_aws/client.py]: is it compatible with all versions or just v3?

**Review by leilacc — DISMISSED** [review]

File: cohere_aws/client.py (hunk 1 above)
Comment (mahjongmen) [reply cohere_aws/client.py]: we should block just for v3

Comment (lfayoux) [reply cohere_aws/client.py]: Do we know the version from the sdk?

Comment (leilacc) [reply cohere_aws/client.py]: I don't think we can enforce the constraint in code, but at least the docstring can say that it's not supported pre-v3

Comment (mahjongmen) [reply cohere_aws/client.py]: @leilacc im guessing there is really no way to know from the ARN or any meta data around what model is being used (like backbone whether its v2/v3). 

I think it's fine to just warn + have in doc-string

**Review by mahjongmen — APPROVED** [review]
LGTM

**Review by leilacc — APPROVED** [review]

**Pull request closed without merging**